src/devtools/views/ActionConfirm.html

```diff
@@ -0,0 +1,14 @@
+<div class="modal fade" id="confirm-action-model">
+	<div class="modal-dialog">
+		<div class="modal-content">
+			<div class="modal-header">
+				<h4 class="modal-title"></h4>
+				<h5 class="modal-subtitle" hidden="true"></h5>
+			</div>
+			<div class="modal-footer">
+				<button class="btn btn-default" id="cancel"></button>
+				<button id="submit" class="btn btn-primary btn-xl"></button>
```

Comment:
there is no `btn-xl` in bootstrap, just remove it, I love resulting style

Reply:
also consider using ids like `modal-submit`, `modal-cancel` to avoid coflicts with page where modal is placed